contexts/userContext.tsx

```diff
@@ -2,6 +2,9 @@ import React, { useContext, useState } from 'react';
 
 const UserContext = React.createContext<any>({
   email: null,
+  id: null,
+  username: null,
+  Trips: [],
   accessToken: null,
 });
```

Comment:
Yessss another one of my planning for future-proofing paying off. Very happy with these.

Reply:
+1